release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3356,6 +3356,46 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+* Previously, when the Node Tuning Operator (NTO) was configured using PerformanceProfiles it would create an ocp-tuned-one-shot systemd service. The systemd service would run prior to kubelet and blocked execution. The systemd service invokes Podman which uses an NTO image, but when the NTO image was not present Podman still tried to fetch the image and it would fail. With this release, support is added for cluster-wide proxy environment variables defined in `/etc/mco/proxy.env`. Now, Podman pulls NTO images in environments which need to use proxies for out-of-cluster connections. (link:https://issues.redhat.com/browse/OCPBUGS-42061[*OCPBUGS-42061*])
+
+* Previously, a change in the ordering of the TextInput parameters for PatternFly v4 and v5 caused the `until` field to be improperly filled and you could not edit it. With this release, the `until` field is editable so you can input the correct information. (link:https://issues.redhat.com/browse/OCPBUGS-41996[*OCPBUGS-41996*])
+
+* Previously, when templates were defined for each failure domain, the installation program required an external connection to download the OVA in vSphere. With this release, the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-41885[*OCPBUGS-41885*])
+
+* Previously, when installing a cluster on bare metal using installer provisioned infrastructure, the installation could time out if the network to the bootstrap virtual machine is slow. With this update, the timeout duration has been increased to cover a wider range of network performance scenarios. (link:https://issues.redhat.com/browse/OCPBUGS-41845[*OCPBUGS-41845*])
+
+* Previously, when a hosted cluster proxy was configured and it used an identity provider (IDP) that had an http or https endpoint, the hostname of the IDP was unresolved before sending it through the proxy. Consequently, hostnames that could only be resolved by the data plane failed to resolve for IDPs. With this update, a DNS lookup is performed before sending IPD traffic through the `konnectivity` tunnel. As a result, IDPs with hostnames that can only be resolved by the data plane can be verified by the Control Plane Operator. (link:https://issues.redhat.com/browse/OCPBUGS-41372[*OCPBUGS-41372*])
+
+* Previously, due to an internal bug, the Node Tuning Operator (NTO) incorrectly computed CPU masks for interrupt and network handling CPU affinity if a machine had more than 256 CPUs. This prevented proper CPU isolation on those machines and resulted in `systemd` unit failures. With this release, the NTO computes the masks correctly.(link:https://issues.redhat.com/browse/OCPBUGS-39377[*OCPBUGS-39377*])
```

Comment:
This phrase _if a machine had more than 256 CPUs_ seems more important and should be earlier in the sentence.
```suggestion
* Previously, due to an internal bug, if a machine had more than 256 CPUs, the Node Tuning Operator (NTO) incorrectly computed CPU masks for interrupt and network handling CPU affinity. This prevented proper CPU isolation on those machines and resulted in `systemd` unit failures. With this release, the NTO computes the masks correctly.(link:https://issues.redhat.com/browse/OCPBUGS-39377[*OCPBUGS-39377*])
```